packages/table/src/Row/Row.tsx

```diff
@@ -6,20 +6,31 @@ import { LGRowData } from '../useLeafyGreenTable';
 import InternalRowWithoutRT from './InternalRowWithoutRT';
 import InternalRowWithRT from './InternalRowWithRT';
 import { RowProps } from './Row.types';
+import { RowContextProvider } from './RowContext';
 
 /**
  * Renders the provided cells
  */
 const Row = <T extends LGRowData>({
   row,
   virtualRow,
+  disabled,
   ...rest
 }: RowProps<T>) => {
-  if (row) {
-    return <InternalRowWithRT row={row} virtualRow={virtualRow} {...rest} />;
-  } else {
-    return <InternalRowWithoutRT {...rest} />;
-  }
+  return (
+    <RowContextProvider disabled={disabled}>
+      {row ? (
+        <InternalRowWithRT
+          row={row}
+          virtualRow={virtualRow}
+          disabled={disabled}
```

Comment:
Do you need to pass disabled if you can get it from the provider?

Reply:
I guess we don't? But it would have to be consumed at every subcomponent of Row then, which feels pretty verbose. The intention here is to introduce the context so it can be consumed from subcomponents defined outside (like Checkbox), not from within Row (since the prop already handles that). If you feel like removing the props from the subcomponents would feel cleaner, I'll make the change!

Reply:
It feels like we now have two ways to pass disabled. The name of the provider, `RowContextProvider` itself makes it seem like it should be consumed inside of `Row`. I feel strongly that we should consume it from the provider since it exists now.

Reply:
Roger that!